interfaces/IWear/include/Wearable/IWear/IWear.h

```diff
@@ -357,8 +360,8 @@ wearable::IWear::getAllActuators() const
     return allActuators;
 }
 
-//template<typename E>
-//inline wearable::VectorOfElementPtr<const E> wearable::IWear::getAllElements() const
+// template<typename E>
+// inline wearable::VectorOfElementPtr<const E> wearable::IWear::getAllElements() const
```

Comment:
Any reason for this whole commented out section to stay here?

Reply:
looks like a left-over, @Yeshasvitvs do you confirm we can remove those lines?

Reply:
Actually, I think that is related to the TODOs here https://github.com/robotology/wearables/blob/master/interfaces/IWear/include/Wearable/IWear/IWear.h#L267-L274. @Yeshasvitvs is it something worth keeping there?

Reply:
I added it as a part of https://github.com/robotology/wearables/pull/105 and the issue related to refactoring of the interfaces is https://github.com/robotology/wearables/issues/106. Feel free to proceed as you see fit @lrapetti

Reply:
Thanks @Yeshasvitvs, I think then we can leave those lines for the time-being in view of completing #106